src/components/Text/Text.type.ts

```
@@ -0,0 +1,30 @@
+
+export interface TextStyleProps {
+  size?: TextSize;
+  textColor?: ColorKeys;
+  responsive?: boolean;
+}
+
+export interface TextStylePropsWithTheme extends TextStyleProps {
+  theme: Theme;
+}
+
+export interface TextCustomProps {}
```

Biome (1.9.4)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)